go-controller/pkg/ovn/base_network_controller_secondary.go

```diff
@@ -412,6 +412,27 @@ func (bsnc *BaseSecondaryNetworkController) removePodForSecondaryNetwork(pod *ka
 		return nil
 	}
 
+	activeNetwork, err := bsnc.getActiveNetworkForNamespace(pod.Namespace)
+	if err != nil {
+		return fmt.Errorf("failed looking for the active network at namespace '%s': %w", pod.Namespace, err)
+	}
+
+	on, networkMap, err := util.GetPodNADToNetworkMappingWithActiveNetwork(pod, bsnc.NetInfo, activeNetwork)
+	if err != nil {
+		bsnc.recordPodErrorEvent(pod, err)
+		// configuration error, no need to retry, do not return error
+		klog.Errorf("Error getting network-attachment for pod %s/%s network %s: %v",
+			pod.Namespace, pod.Name, bsnc.GetNetworkName(), err)
+		return nil
```

Comment:
return nil was intentional? we used to retry the deletion if we failed to get this